src/main/java/org/tron/core/db/Manager.java

```diff
@@ -97,6 +97,8 @@
 @Component
 public class Manager {
 
+  public static ThreadLocal<Boolean> isProcessBLock = new ThreadLocal<>();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make this "public static isProcessBLock" field final ([squid:S1444](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1444))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1471&issues=AWW0g65bashydt7kZwXr&open=AWW0g65bashydt7kZwXr)

[ISSUE_KEY]: AWW0g65bashydt7kZwXr